test/singlenode_test.go

```diff
@@ -33,6 +33,70 @@ func TestClientCouldConnect(t *testing.T) {
 	})
 }
 
+func TestClientConnectDuplicate(t *testing.T) {
+	if testing.Short() {
+		t.SkipNow()
+		return
+	}
+
+	_, finish := setupSingleNodeTest("TestClientCouldConnect", testRoom)
+	defer finish()
+
+	c1 := createRTCClient("c1", defaultServerPort, nil)
+
+	opts := &testclient.Options{
+		Publish: "duplicate_connection",
+	}
+
+	c1Dup := createRTCClient("c1", defaultServerPort, opts)
```

Comment:
I think this isn't accurate enough, because it'll connect with a new token. we should be connecting with the same token but with the publish parameter

Reply:
will update it